pkg/render/manager.go

```diff
@@ -256,6 +260,21 @@ func (c *managerComponent) managerVolumes() []v1.Volume {
 				},
 			},
 		},
+		{
+			// We only want to mount the cert, not the private key to es-proxy to establish a connection with voltron.
+			Name: ManagerTLSSecretCertName,
```

Comment:
Is it correct for this to be present all the time? (In other words is this needed for standalone clusters?)